### PR TITLE
Milestone 3:  Set up associations

### PR DESCRIPTION
**Pull request by Efiamotu-1**
In this PR, the following were implemented:-

>- Created a relationship between books and rentals
>- Created a relationship between students and a classroom

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
```
Comment (tufoinnkuo10) [inline classroom.rb:4]: - [x] Great work. Kindly add students as `attr_reader`

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Rental
+  def initialize(date, person, book)
```
Comment (tufoinnkuo10) [inline rental.rb:4]: - [x] Great job. please, kindly ensure Getters and Setters for @date, @book and @person instance variables should be present. (`attr_accessor`)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
## Hi @Efiamotu-1 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: classroom.rb (hunk 1 above)
Comment (Efiamotu-1) [reply classroom.rb:4]: @tufoinnkuo10 you didn't specify the reason why i should add this, could you please provide more context. Thank you

File: rental.rb (hunk 2 above)
Comment (Efiamotu-1) [reply rental.rb:4]: i have implement this now, i didn't  realise it wasn't added before my first submission. Thank you

**Review by prolajumokeoni — APPROVED** [review]
Hi @Efiamotu-1 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**